config/initializers/component_paths.rb

```diff
@@ -0,0 +1,3 @@
+ActiveSupport.on_load :action_controller do
+  append_view_path Rails.root.join("app", "components")
```

Comment:
this could be pulled out of some global `ActionView::Component` config

Reply:
Hmm. 

I'm curious if there might be a way to pull this off without loading the component templates into the global view paths, as my gut says it might not be a good idea to expose those templates outside the components themselves.